developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -132,6 +132,27 @@ All rake tasks are available with the following command:
 bundle exec rake -T
 ....
 
+=== Running tests in Foreman
+Make sure to run tests from the Foreman directory.
+
+In order to run rubocop test, use the following command:
+[source, shell]
+....
+bundle exec rubocop [<path_to_file>]
+....
+
+To run Foreman's tests:
+[source, shell]
+....
+bundle exec rake test [TEST=<path_to_file>]
```

Comment:
Should we add the compile line like we have in plugin tests?
```
bundle exec rake webpack:compile # only needed if you have integration tests that uses JS
bundle exec rake test:<PLUGIN_NAME>
```

Reply:
Thanks Maria! You are right, I'll add it.